src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj

```diff
@@ -340,6 +340,7 @@
     <None Include="SessionRecords\**\*.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Compute.Test.ScenarioTests.AEMExtensionTests\TestAEMExtensionAdvancedWindowsMD.json" />
```

Comment:
You need to copy this to the output directory.  The previous Include *might* cover this, but it might not.